test/qpy_compat/process_version.sh

```diff
@@ -14,9 +14,23 @@
 set -e
 set -x
 
+# version is the source version, this is the release with which to generate
+# qpy files with to load with the version under test
 version=$1
 parts=( ${version//./ } )
-if [[ ${parts[1]} -lt 18 ]] ; then
+# qiskit_version is the version under test, We're testing that we can correctly
+# read the qpy files generated with source version with this version.
+qiskit_version=`../qiskit_venv/bin/python -c "import qiskit;print(qiskit.__version__)"`
+qiskit_version_pars = ( ${qiskit_version//./ } )
```

Comment:
There's a naming problem here - we call the variable `qiskit_version_pars` (lol) but then use it as `qiskit_parts`.

Reply:
I fixed this in: https://github.com/Qiskit/qiskit/pull/11101/commits/1c87c502c228942723fd7299af648881687de35e (I should have split it out into a second commit but forgot and just did `git add` without the `-p`)